src/templates/framework/nav.phtml

```diff
@@ -1,7 +1,14 @@
 <nav>
    <ul>
-      <li><a class="fun" href="#">About Me</a></li>
-      <li><a href="#">My Projects</a></li>
-      <li><a href="#">My Resume</a></li>
+      <?php foreach ($this->getPacketData('navIndexRows') as $i => $navRow) :
+         $class = $navRow['isFun'] ? 'fun' : '';
+      ?>
+         <li>
+            <a class="<?= $class ?>" href="<?= $navRow['url'] ?>">
+               <?= $navRow['display'] ?>
+            </a>
+         </li>
+      <?php endforeach ?>
+      <li><a href="#resume">My Resume</a></li>
```

Comment:
Not quite sure how we'd handle "files" in the nav programmatically. It doesn't really work with my idea of the `page_index`.

Probably splitting out the `nav` values and the `page_index` data is the best bet if I wanted this feature. Add a "type" column to `nav` something like `nav_type - [external-link, page_index]`.

Maybe also just throw a redirect in the `doSomething` page logic...

Doesn't really matter b/c we can hardcode it but it's annoying to have this be different.